official/vision/beta/projects/yolo/modeling/backbones/darknet.py

```diff
@@ -383,9 +384,11 @@ def __init__(
       max_level=5,
       width_scale=1.0,
       depth_scale=1.0,
+      use_reorg_input = False, 
```

Comment:
what is that?

Reply:
used in some of the newer models, allows you to pass in an input of say 1080 by 1080 then resample the RGB image to create and stack together 12 images of 1080/2, so input goes from [None, 1080, 1080, 3] to  [None, 540, 540, 12], the inputs are resamples in such a way that spatial information is retained.

Reply:
addressed